<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Pref::TAdvanced</class>
 <widget class="QWidget" name="Gui::Pref::TAdvanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="advanced_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>&amp;Advanced</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>16</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="monitor_aspect_icon">
           <property name="text">
            <string notr="true">icon</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel1_8">
           <property name="text">
            <string>&amp;Monitor aspect:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>monitoraspect_combo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Gui::Pref::TComboBox" name="monitoraspect_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>190</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="idx_check">
         <property name="text">
          <string>Rebuild &amp;index if needed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lavf_demuxer_check">
         <property name="text">
          <string>Use the la&amp;vf demuxer by default</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="repaint_video_background_check">
         <property name="text">
          <string>Repaint the backgroun&amp;d of the video window</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mplayer_crashes_check">
         <property name="text">
          <string notr="true">R&amp;eport PLAYER crashes</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>C&amp;orrect PTS:</string>
           </property>
           <property name="buddy">
            <cstring>correct_pts_combo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Gui::Pref::TTristateCombo" name="correct_pts_combo"/>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>R&amp;un the following actions every time a file is opened. The actions must be separated with spaces:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>actions_to_run_edit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="actions_to_run_edit"/>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="colorkey_label">
           <property name="text">
            <string>&amp;Colorkey:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>colorkey_view</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="colorkey_view">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="changeButton">
           <property name="text">
            <string>C&amp;hange...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>131</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string notr="true">O&amp;ptions for PLAYER</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>16</number>
       </property>
       <item>
        <widget class="QLabel" name="options_info_label">
         <property name="text">
          <string notr="true">Here you can pass extra options to PLAYER.
Write them separated by spaces.
Example: -flip -nosound</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1_3">
           <property name="text">
            <string>&amp;Options:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>mplayer_args_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mplayer_args_edit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel4_2">
         <property name="text">
          <string>You can also pass additional video filters.
Separate them with &quot;,&quot;. Do not use spaces!
Example: scale=512:-2,mirror</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel2_3">
           <property name="text">
            <string>V&amp;ideo filters:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>mplayer_vfilters_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mplayer_vfilters_edit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel5_2">
         <property name="text">
          <string>And finally audio filters. Same rule as for video filters.
Example: extrastereo,karaoke</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel3_3">
           <property name="text">
            <string>Audio &amp;filters:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>mplayer_afilters_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mplayer_afilters_edit"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::Pref::TComboBox</class>
   <extends>QComboBox</extends>
   <header>gui/pref/combobox.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::Pref::TTristateCombo</class>
   <extends>QComboBox</extends>
   <header>gui/pref/tristatecombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>monitoraspect_combo</tabstop>
  <tabstop>idx_check</tabstop>
  <tabstop>lavf_demuxer_check</tabstop>
  <tabstop>repaint_video_background_check</tabstop>
  <tabstop>mplayer_crashes_check</tabstop>
  <tabstop>correct_pts_combo</tabstop>
  <tabstop>actions_to_run_edit</tabstop>
  <tabstop>colorkey_view</tabstop>
  <tabstop>changeButton</tabstop>
  <tabstop>mplayer_args_edit</tabstop>
  <tabstop>mplayer_vfilters_edit</tabstop>
  <tabstop>mplayer_afilters_edit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
